tests/functional_tests/t0_main/grpc/t3_use_artifact.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import os
+
+import wandb
+
+# Temporary environment variable for testing grpc service mode
+os.environ["WANDB_SERVICE_TRANSPORT"] = "grpc"
```

Comment:
why do you test the grpc mode and not the socket one? (by default we use socket mode, you could test both)

Reply:
Yep, there are other tests that handle socket mode

Reply:
cool thanks such a complete fix!!!